Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Terms/CreateTermHandlerTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using AutoMapper;
+using MediatR;
+using Moq;
+using Streetcode.BLL.DTO.Streetcode.TextContent;
+using Streetcode.BLL.MediatR.Streetcode.Term.Create;
+using Streetcode.DAL.Entities.Streetcode.TextContent;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+
+using Xunit;
+
+namespace Streetcode.XUnitTest.MediatRTests.StreetCode.Terms
+{
+	public class CreateTermHandlerTests
+	{
+		private Mock<IRepositoryWrapper> _mockRepository;
+		private Mock<IMapper> _mockMapper;
+
+		public CreateTermHandlerTests() 
+		{
+			_mockMapper = new();
+			_mockRepository = new();
+		}
+
+		[Fact]
+		public async Task ShouldReturnSuccessfully_TypeIsCorrect() {
+			//Arrange
+			_mockRepository.Setup(x => x.TermRepository.Create(GetTerm()));
+			_mockRepository.Setup(x => x.SaveChangesAsync()).ReturnsAsync(1);
+
```

Comment:
please avoid using magical numbers, it is a good practice to move them to attribute [InlineData()]